3_predictions/out/model_evaluation.csv

```diff
@@ -2,6 +2,6 @@
 "rgcn_subset",0,1.24,1.3,1.08,1.2,1.59,10256
 "rgcn2_full_subset",-0.18,1.47,1.66,1.42,1.19,2.02,10256
 "rgcn2_subset",-0.05,1.23,1.46,1.04,1.16,1.71,10256
-"sntemp_subset",0.97,3.7,2.62,3.48,5.65,2.48,10256
+"sntemp_subset",0.97,3.66,2.57,3.45,5.58,2.6,10256
 "rgcn2_full_full",0.09,1.76,2.02,1.51,1.8,2.24,43093
-"sntemp_full",1.58,4.08,4.6,3.49,6.17,4.13,43093
+"sntemp_full",1.59,5.02,6.03,4.38,6.66,5.04,43093
```

Comment:
sntemp subset is better with gridmet drivers but sntemp full network is worse - should look into this more

Reply:
[test period is slightly better using Gridmet ](https://github.com/USGS-R/delaware-model-prep/issues/74#issuecomment-708001229)